### PR TITLE
FlightTaskManualStabilized: allow attitude control with 0 throttle

### PR DESCRIPTION
**Pull request by bkueng**
With minimum throttle set to 0, it was not possible to control the attitude
anymore at 0 throttle, the vehicle would just stay level.

@MaEtUgR @Stifael do you have another way of fixing it? Other flight tasks might be affected too.

### CODE REVIEW

**Comment by Stifael** [comment]
is your minimum throttle (https://github.com/PX4/Firmware/blob/master/src/modules/mc_pos_control/mc_pos_control_params.c#L101) 0? Because otherwise the throttle curve (https://github.com/PX4/Firmware/blob/master/src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp#L147) will scale the throttle to non 0 value

**Comment by Stifael** [comment]
Ok. 
> Other flight tasks might be affected too.
Does it make sense that minimum thrust is set to 0 for other Flighttasks? By the way, Stabilized uses and other minmum throttle parameter (https://github.com/PX4/Firmware/blob/master/src/modules/mc_pos_control/mc_pos_control_params.c#L53 and https://github.com/PX4/Firmware/blob/master/src/modules/mc_pos_control/PositionControl.hpp#L190) and hence it only applies to stabilized.

File: src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp [hunk 1]
```diff
@@ -123,9 +123,10 @@ void FlightTaskManualStabilized::_updateThrustSetpoints()
 
 	/* The final thrust setpoint is found by rotating the scaled unit vector pointing
 	 * upward by the Axis-Angle.
+	 * Make sure that the attitude can be controlled even at 0 throttle.
 	 */
 	Quatf q_sp = AxisAnglef(v(0), v(1), 0.0f);
-	_thrust_setpoint = q_sp.conjugate(Vector3f(0.0f, 0.0f, -1.0f)) * _throttle;
+	_thrust_setpoint = q_sp.conjugate(Vector3f(0.0f, 0.0f, -1.0f)) * math::max(_throttle, 0.0001f);
```
Comment (Stifael) [inline src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp]: With the current FlightTask structure, I think that would be the easiest fix. If you think that the flight performance is different now, then we probably need to add a special case when the parameter is set to 0

Comment (bkueng) [reply src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp]: > If you think that the flight performance is different now, then we probably need to add a special case when the parameter is set to 0

What do you mean by different? With this fix it flies as expected. I was just wondering if it's a more general problem that needs to be fixed in other FlightTasks as well.

Comment (Stifael) [reply src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp]: I think we are talking at cross purposes.
There are two things i wanted to say: 
1. I don't have a better solution as the one you have provided. The only other option would be to create a special case, but I prefer your solution as long as it works.
2. other flighttasks might be effected: please look at https://github.com/PX4/Firmware/pull/10545#issuecomment-423890137

Comment (bkueng) [reply src/lib/FlightTasks/tasks/ManualStabilized/FlightTaskManualStabilized.cpp]: Ah sorry I didn't see your edit. All good then.

**Pull request merged**